todo-list/src/hooks/todos/useTodo.ts

```diff
@@ -59,3 +70,60 @@ export function useAddTodo() {
     error: mutation.error,
   };
 }
+
+export function useUpdateTodo() {
+  const updatedTodoStore = useTodoStore((state) => state.updateTodo);

```

Comment:
Wrong and confuse naming.

Read more about naming convention here: https://github.com/airbnb/javascript?tab=readme-ov-file#naming-conventions